src/foundry/client/pixi/placeables/drawing.d.mts

```diff
@@ -104,7 +104,7 @@ declare global {
      */
     static readonly SHAPE_TYPES: (typeof foundry.data.ShapeData)["TYPES"];
 
-    override get bounds(): Rectangle;
+    override get bounds(): PIXI.Rectangle;
```

Comment:
The parent class return `PIXI.Rectangle`. I did make sure to check that no subclass actually returned `Rectangle` (the Foundry class)